content/akeneo-event-platform/concepts.md

```diff
@@ -52,13 +48,13 @@ The statuses for a subscription are:
 | `active` | The destination will receive notifications for events monitored by the subscription |
 | `deleted` | The subscription is inactive and cannot be reactivated |
 | `suspended` | The subscription has been deactivated by the event platform itself due to excessive errors, or manually by the user. However, you can resume it. Suspending a subscription stops all events from being sent to it. Events are not saved and are lost until the subscription is resumed |
-| `revoked` | The subscription has been automatically deactivated because the destination responded with an HTTP 404. You can manually reactivate it. Suspending a subscription halts all event deliveries to this subscription. Events are not saved and are lost until the subscription is resumed |
+| `revoked` | The subscription has been automatically revoked because the connection or the app linked to the subscriber were removed from the PIM |
```

Comment:
```suggestion
| `revoked` | The subscription has been automatically revoked because the connection or the app linked to the subscriber was removed from the PIM |
```